apps/avatax/src/modules/calculate-taxes/use-case/calculate-taxes.use-case.test.ts

```diff
@@ -127,6 +131,7 @@ describe("CalculateTaxesUseCase", () => {
 
     instance = new CalculateTaxesUseCase({
       configExtractor: MockConfigExtractor,
+      discountsStrategy: MockDiscountStrategy,
```

Comment:
1. add test to UseCase to check strategy
2. Dont inject strategy, we decided that UseCase should make its own decision which strategy to use, so we shouldn't inject this

Reply:
1. I decided not to do tests on usecase as we need to mock whole Avatax client - I added todo instead to implement tests when we allow injecting client to usecase